credentials/credentials_github_app.go

```diff
@@ -0,0 +1,43 @@
+package credentials
+
+import (
+	"fmt"
+)
+
+// GithubAppCredentials represents credentials composed of a private key, username and passphrase
+type GithubAppCredentials struct {
+	Base       `mapstructure:",squash"`
+	AppID      int    `mapstructure:"app_id"`
+	PrivateKey string `mapstructure:"private_key"`
+	Owner      string `mapstructure:"owner"`
+}
+
+// NewGithubAppCredentials instantiates a GithubAppCredentials struct
+func NewGithubAppCredentials() *GithubAppCredentials {
+	cred := &GithubAppCredentials{}
+	cred.CredType = "Github App"
+	return cred
+}
+
+// ToString prints out the content of a GithubAppCredentials struct.
+// showSensitive bool has not effect. It is provided to satisfy the interface.
```

Comment:
Leftover from copy paste?

Reply:
No, I added this comment, but had to remove the variable name to avoid a warning about variable not being used.
I'm using it now: 7ae992f